<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de Visi√≥n Artificial</string>
  </property>
  <widget class="QFrame" name="imageFrameS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="imageFrameD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Capture</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="colorButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Image</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load from File</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to File</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="copyChannelsButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy channels</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameCheck">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="rCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="gCheck">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bCheck">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="copyWindowButton">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy window</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resizeButton">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize window</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="enlargeButton">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enlarge window</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
